receiver/snmpreceiver/scraper.go

```diff
@@ -229,7 +224,7 @@ func (s *snmpScraper) indexedDataToMetric(
 	var resourceKey string
 	// If we only have scalar resource attributes, we don't need multiple resources
 	if len(resourceAttributes) == len(columnOIDScalarResourceAttributeValues) {
-		resourceKey = getResourceKey(resourceAttributeNames, "0")
+		resourceKey = getResourceKey(resourceAttributeNames, "")
```

Comment:
would indexString already be an empty string here (in the case of scalar resource attributes) or no? if so we could get rid of the if/else

Reply:
Not in this case but good call.